package.json

```diff
@@ -179,7 +180,7 @@
     "start-js": "react-scripts start",
     "start": "npm-run-all -p watch-css serve:docs",
     "start-deprecated-demo": "npm-run-all -p watch-css start-js",
-    "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!(lodash-es|jest*)/)\" , --env=jsdom-fourteen --setupFiles ./src/setupTests.js",
+    "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!(lodash-es|jest*)/)\" , --env=jsdom --setupFiles ./src/setupTests.js --resetMocks false",
```

Comment:
Breaking change in upgrading Jest from v24 to v26 requires setting `--resetMocks=false`. Additionally, the default version of JSDOM is now v16, so we don't need to specify jsdom-fourteen anymore